src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -724,11 +791,9 @@ class GmailRouteView {
       .map((part) => part.substring(1));
   }
 
-  _getThreadContainerElement: () => HTMLElement | null | undefined = once(
-    () => {
-      return GmailElementGetter.getThreadContainerElement();
-    },
-  );
+  _getThreadContainerElement() {
```

Comment:
We should cu this method in its entirety and just use the `GmailElementGetter` equivalent directly.

Reply:
ill keep existing method because it is being used in few more places, want to limit amount of changes for this fix